smart-py/proxy.py

```diff
@@ -0,0 +1,42 @@
+import smartpy as sp
+
+class Proxy(sp.Contract):
```

Comment:
Add a comment on the purpose of the proxy.

Reply:
done